src/event-topics/event-subscription-topic-accountingintegration.markdown

```diff
@@ -10,41 +10,170 @@ layout: reference
-`listMapping.deleted`|This event will be published after the customer has removed the mapping between the SAP Concur systems list and the ERP list name.|The subscriber may remove the list items from SAP Concur.
-`data.resetRequested`|This event will be published when the customer has requested to reset the ERP data cache.|The subscriber may reset the ERP data cache.
-`data.statusUpdated`|This event will be published after the completion of the processing of the ERP data.|The subscriber may query for the status details.
+|Event Type|Condition that triggers event|Recommended Partner Action|
+|---|---|---|
+|`listMapping.created`| This event will be published after the customer administrator has mapped the SAP Concur list with the ERP list name. (Professional Edition Note: The customer admin must first import the list, then map them in the `Forms and Fields` interface. Multiple fields can be linked to a single list. Only the first mapping of a list will trigger this event) | This is a trigger telling the Partner that it's time for the ERP data to be synchronized (Via the List Items API) with SAP Concur data.|
+|`listMapping.deleted`| This event will be published after the customer administrator has removed the mapping between the SAP Concur list and the ERP list name. (Professional Edition Note: As multiple-fields can be mapped to the same list - this event will only trigger when all fields are unmapped from a given list) | The indicated list is no longer mapped and allows the Partner to do it's own data cleanup. This should also stop the indicated list data from being synchronized.|
+|`data.resetRequested`| This event will be published when the customer administrator has requested to reset the ERP data cached in SAP Concur data-center. | Data should be resynchronized to SAP Concur systems.|
+|`data.statusUpdated`| This event will be sent to notify the partner that the data they sent to SAP Concur systems via the PATCH action with the `Accounts`, `Lists` and `Vendors` APIs has been processed on SAP Concur side. One notification is sent for each PATCH request that the partner performs. | The status may be interesting for the Partner to perform actions such as notification, logging, cleanup, etc.|
 
 ## <a name="schema"></a>Schema
 
 ### <a name="listMappingCreated"></a>Schema for Event `listMapping.created` and `listMapping.deleted`
 
-Name|Type|Format|Description
----|---|---|---
-`data`|`array`|[`ListMapping`](/api-reference/accounting-integration/v4.accountingintegration-schema.html#list-mapping)|List of ERP list to SAP Concur list mappings.
+|Name|Type|Format|Description|
+|---|---|---|---|
+| `data` | `array` | [`ListMapping`](/api-reference/accounting-integration/v4.accountingintegration-schema.html#list-mapping) | List of ERP list to SAP Concur list mappings (See examples below) |
```

Comment:
Does my suggested language of "List mappings between the ERP and SAP Concur systems." not work? Because "list mappings" is also not an approved trademark descriptor so this still needs to change. Besides, it may be confusing to ESL readers when you have "list" in the same sentence three times.

Reply:
This is a list of list-mappings - which is awkward. See if my new line works for you.